examples/headsail-bsp/Cargo.toml

```diff
@@ -49,3 +50,8 @@ required-features = ["rt"]
 name = "alloc"
 path = "examples/alloc.rs"
 required-features = ["alloc", "rt", "sdram"]
+
+[[example]]
+name = "timer0"
+path = "examples/timer0.rs"
+required-features = ["rt"]
```

Comment:
This entry implies that there should be an example under project `headsail-bsp` at location `examples/headsail-bsp/examples/timer0.rs". In reality, the example is located under project `hello-hpc` at `examples/hpc/hello-hpc/examples/timer0.rs`.

Pick one:

- Move the exercise from hello-hpc/examples to headsail-bsp/examples (**preferred**)
- Move the [[example]] entry from headsail-bsp/Cargo.toml to hello-hpc/Cargo.toml

Reply:
Done, moved timer0.rs to headsail-bsp/examples.